bundles/org.openhab.core/src/main/java/org/openhab/core/addon/AddonI18nLocalizationService.java

```diff
@@ -53,6 +53,14 @@ public AddonInfo createLocalizedAddonInfo(Bundle bundle, AddonInfo addonInfo, @N
         String name = addonI18NUtil.getName(bundle, addonInfoUID, addonInfo.getName(), locale);
         String description = addonI18NUtil.getDescription(bundle, addonInfoUID, addonInfo.getDescription(), locale);
 
+        if (name == null) {
```

Comment:
IMO `AddonI18NUtil` should be fixed that the return values are `@NonNull`, because we supply a default value that is non-null. In `.getDescription` the parameter annotation should be changed, too.

Reply:
Better now?